DNN Platform/Library/Entities/Urls/AdvancedUrlRewriter.cs

```diff
@@ -2339,6 +2339,7 @@ private static bool IgnoreRequestForInstall(string physicalPath, string refererP
         private static bool IgnoreRequestForWebServer(string requestedPath)
         {
             if (requestedPath.ToLowerInvariant().IndexOf("synchronizecache.aspx", StringComparison.Ordinal) > 1
+                || requestedPath.ToLowerInvariant().IndexOf("synchronizeoutputcache.aspx", StringComparison.Ordinal) > 1

```

Comment:
Let's put the `synchronizeoutputcache.aspx` as a static readonly field of `MemoryCacheSynchonizationHandler`

Reply:
Also `requestedPath.IndexOf("synchronizeoutputcache.aspx", StringComparison.OrdinalIgnoreCase)` will be cleaner